targets/extension

```diff
@@ -19,7 +19,13 @@ fi
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install x11-utils xclip
+EXT_APPS=""
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    EXT_APPS="x11-misc/xclip appres luit xdpyinfo xdriinfo xev xfd xfontsel xkill xlsatoms xlsclients xlsfonts xmessage xprop xvinfo xwininfo"
```

Comment:
no sense in setting a variable if you're just going to call install.  Just run install directly.